build/build-report.js

```diff
@@ -42,6 +42,12 @@ async function buildFlowReport() {
       commonjs(),
       typescript({
         tsconfig: 'flow-report/tsconfig.json',
+        // Plugin struggles with custom outDir, so revert it from tsconfig value
+        // as well as any options that require an outDir is set.
+        outDir: null,
+        composite: false,
+        emitDeclarationOnly: false,
+        declarationMap: false,
```

Comment:
these override the values in the `tsconfig` file. Another common pattern is to have a `tsconfig.rollup.json` file alongside the `tsconfig.json` with these overrides in there instead. I'm fine with either approach, so whatever you'd prefer @adamraine

Reply:
I like having the overrides here. LGTM.